frontend/src/Layout.tsx

```diff
@@ -0,0 +1,22 @@
+import { Outlet } from "react-router-dom";
+import { Header } from "./components/organisms/Header";
+import { CSSProperties } from "react";
+
+const Styles: { [key: string]: CSSProperties } = {
+  main: {
+    marginTop: "80px",
+  },
+};
+
+const Layout = () => {
+  return (
+    <div>
+      <Header />
+      <main style={Styles.main}>
+        <Outlet />
+      </main>
+    </div>
```

Comment:
このトップレベルのdivは特別スタイル当てるわけじゃないならReact.Fragmentでもいいかも
これだと以下みたいにレンダリングされて、純粋なHTML構造としてはちょっと歪な感覚がする

```html
<html>
  <body>
    <div> <--- このdivが気になる
      <header>
      </header>
      <main>
        <Outlet />
      </main>
    </div>
  </body>
</html>
```

```suggestion
    <>
      <Header />
      <main style={Styles.main}>
        <Outlet />
      </main>
    </>
```

Reply:
[このコミット](b935a3a0e9f3fa9ae740295a44a9764b1167f3df)で対応